handlers/handlers.go

```diff
@@ -195,11 +212,17 @@ func HandleSessionRequest(
 	}
 }
 
+// AuthentictedRequestHandlerType is the function signature for all authenticated
```

Comment:
I believe you said this elsewhere, but I suppose repeating ourselves isn't necessarily wrong